### PR TITLE
fix(ql): Add moduleMember to indent file

### PR DESCRIPTION
**Pull request by pwntester**

### CODE REVIEW

File: queries/ql/indents.scm [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  (moduleMember)
```
Comment (theHamsta) [inline queries/ql/indents.scm:2]: could you add a test for this?

Comment (pwntester) [reply queries/ql/indents.scm:2]: I did, thanks!

Comment (pwntester) [reply queries/ql/indents.scm:2]: Actually, thanks to the tests I realize that the indentation is not yet ok. Will fix and add more tests

Comment (pwntester) [reply queries/ql/indents.scm]: Ok, seems like its working now! BTW, I tried running the `scripts/run_tests.sh` script but it seems to be missing the `plugin/plenary.vim` file. Is that correct?

**Review by theHamsta — APPROVED** [review]

File: tests/indent/ql/qlpack.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+library: false
+name: test/test
+version: 0.0.1
+dependencies:
+  codeql/go-all: '*'
```
Comment (theHamsta) [inline tests/indent/ql/qlpack.yml:6]: what does this file do?

Comment (pwntester) [reply tests/indent/ql/qlpack.yml:6]: That file tells the CodeQL interpreter where to get the libraries so it does not throw any errors on the `import go` statement. But I can simplify it and remove this file and the import line

**Review by theHamsta — APPROVED** [review]

**Pull request merged**